### ISSUE TITLE
Formatting problems in xChar and xFeeds pages when replying to messages in telegram channel using xSync service

### ISSUE THREAD

**Issue by pseudoyu** [Crossbell-Box/OperatorSync#61]
## Problem Description

I use the xSync service to sync all messages in my Telegram Channel "Yu's Life" ([t.me/psedoyulife](https://t.me/pseudoyulife)) to the Crossbell Blockchain.

The xSync service works well for regular messages, but when I reply to(quote) a message in the channel, there are some formatting issues in the synced content displayed on xChar and xFeeds.

The original content is

```plaintext
其实村上的随笔或是文集算是很好读的那种，言辞带些犀利却也同样带着些娓娓道来的真诚与别样韵味，像是对话，却也时常被埋在其中似乎带着些雕琢的幽默感逗得不禁莞尔。

总共大概花了三个月才真正看完，作为手头上为数不多的纸质书，更多的是想起时随便翻翻。

第一次去猫舍的时候虽然心动，却也本没有太下定决心那么快承担这样的责任，而路上随手翻起的一页里写道

“当时我们过着非常节俭的斯巴达式的生活。家里既没有电视也没有收音机，甚至连一只闹钟都没有。也几乎没有取暖设备，寒夜里只好紧紧搂着家里养的几只猫咪睡觉。猫咪们也使劲往我们身上贴过来。“

短短几句，却构成了我当下把猫带回家的最大动力。也谈到了许多，关于爱好、写作目的、创作思路与平衡，都是近两个月慢慢沉静下来才真正能够稍理解些的。
```

The displayed synced content is

```html
   <p><span class="emoji">📖</span> 读过我的职业是小说家 #douban #read  https://book.douban.com/subject/26889236/</p>
                            </blockquote><p>其实村上的随笔或是文集算是很好读的那种，言辞带些犀利却也同样带着些娓娓道来的真诚与别样韵味，像是对话，却也时常被埋在其中似乎带着些雕琢的幽默感逗得不禁莞尔。<br><br>总共大概花了三个月才真正看完，作为手头上为数不多的纸质书，更多的是想起时随便翻翻。<br><br>第一次去猫舍的时候虽然心动，却也本没有太下定决心那么快承担这样的责任，而路上随手翻起的一页里写道<br><br>“当时我们过着非常节俭的斯巴达式的生活。家里既没有电视也没有收音机，甚至连一只闹钟都没有。也几乎没有取暖设备，寒夜里只好紧紧搂着家里养的几只猫咪睡觉。猫咪们也使劲往我们身上贴过来。“<br><br>短短几句，却构成了我当下把猫带回家的最大动力。也谈到了许多，关于爱好、写作目的、创作思路与平衡，都是近两个月慢慢沉静下来才真正能够稍理解些的。</p>
```

## Related Account

The Crossbell address affected by this issue is 0xb461c1521eE9D96a9C950337f0851b79bD66CAe1.

## Screenshots

![CleanShot 2023-02-03 at 12 17 12@2x](https://user-images.githubusercontent.com/69753389/216511312-9874f732-ae28-4b43-9080-29aee3105bb3.png)

![CleanShot 2023-02-03 at 12 19 47@2x](https://user-images.githubusercontent.com/69753389/216511618-53b9c3aa-8092-45c8-8a0e-0af334fc9bb1.png)

**Comment by Candinya** [comment]
Should be fixed in 024caa8e2a9e64cbc37f636151da27d8dcfded79 89d67edaa1f2bfe883803bb3ddbceb694af0e5e1 , production soon 👀

**Comment by pseudoyu** [comment]
> Should be fixed in [024caa8](https://github.com/Crossbell-Box/OperatorSync/commit/024caa8e2a9e64cbc37f636151da27d8dcfded79) [89d67ed](https://github.com/Crossbell-Box/OperatorSync/commit/89d67edaa1f2bfe883803bb3ddbceb694af0e5e1) , production soon 👀

Thank you for your efforts!!! Will try when in prod.☺️
